pkg/model/gcemodel/autoscalinggroup.go

```diff
@@ -251,11 +241,11 @@ func (b *AutoscalingGroupModelBuilder) buildInstanceTemplate(c *fi.CloudupModelB
 
 			t.ServiceAccounts = append(t.ServiceAccounts, b.LinkToServiceAccount(ig))
 
-			// labels, err := b.CloudTagsForInstanceGroup(ig)
```

Comment:
lol, thanks for fixing ... I'll have to go splunking to figure out why we commented this out and special-cased it :-)

Reply:
Yeah I tried but it's hard to follow the change history because a few of the recent changes were just formatting.